### PR TITLE
Fix rendering of math text

### PR DESCRIPTION
**Pull request by objorke**
To solve #569 and #448

### CODE REVIEW

**Comment by objorke** [comment]
Does someone else want to review this? Otherwise I merge tomorrow :-)

File: Source/Examples/ExampleLibrary/ExampleInfo.cs [hunk 1]
```diff
@@ -110,7 +110,7 @@ public string Code
         {
             get
             {
-                return this.PlotModel.ToCode();
+                return this.PlotModel != null ? this.PlotModel.ToCode() : null;
```
Comment (objorke) [inline Source/Examples/ExampleLibrary/ExampleInfo.cs]: null reference exception in the "PlotModel examples"

File: Source/OxyPlot/Rendering/RenderContext/MathRenderingExtensions.cs [hunk 2]
```diff
@@ -205,35 +212,37 @@ static MathRenderingExtensions()
...
-            double subscriptFontSize = fontSize * SubSize;
+            var superscriptFontSize = fontSize * SuperSize;
+            var subscriptFontSize = fontSize * SubSize;
 
             Func<double, double, string, double, OxySize> drawText = (xb, yb, text, fSize) =>
                 {
                     if (!measureOnly)
                     {
-                        rc.DrawText(new ScreenPoint(xb, yb), text, textColor, fontFamily, fSize, fontWeight, angle);
+                        var xr = x + ((xb - x + dx) * cosAngle) - ((yb - y + dy) * sinAngle);
```
Comment (objorke) [inline Source/OxyPlot/Rendering/RenderContext/MathRenderingExtensions.cs]: calculate rotated x and y

Comment (objorke) [inline Source/OxyPlot/Rendering/RenderContext/MathRenderingExtensions.cs]: measurements are done in unrotated coordinates

File: Source/OxyPlot/Rendering/RenderContext/MathRenderingExtensions.cs [hunk 3]
```diff
@@ -322,17 +301,15 @@ static MathRenderingExtensions()
...
-                maximumY = Math.Max(currentY, maximumY);
+                maximumX = Math.Max(currentX + size2.Width, maximumX);
+                maximumY = Math.Max(currentY + size2.Height, maximumY);
                 minimumX = Math.Min(currentX, minimumX);
                 minimumY = Math.Min(currentY, minimumY);
+
+                currentX = maximumX;
             }
 
             return new OxySize(maximumX - minimumX, maximumY - minimumY);
```
Comment (objorke) [inline Source/OxyPlot/Rendering/RenderContext/MathRenderingExtensions.cs]: the returned size is not rotated, consistent with the other text measurements

**Comment by tibel** [comment]
Looks good to me.

**Pull request merged**